lib/logdna-rails.rb

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-# encoding: utf-8
+#encoding: utf-8
```

Comment:
This *should* have a space after '#'

Reply:
This comment still needs addressing. Should probably just revert changes in this file.

Reply:
I can't revert the change bc the name of the should be changed. I'll add the space for the first comment as well for consistency.